### ISSUE TITLE
Launching via Trenchbroom on macOS fails to lock cursor and unable to fire

### ISSUE THREAD

**Issue by dill0wn** [Novum/vkQuake#414]
This is a strange one. Launching vkquake normally works. However, if I configure vkquake through TrenchBroom's engine UI, and try to launch the map i'm editing, it shows my mouse cursor and i'm unable to left click to fire.

**Comment by dill0wn** [comment]
I found the issue. It was a bad configuration in TrenchBroom. 

For quakespasm, I had needed to set the launch args to 
```
-basedir ${GAME_DIR_PATH} -game ${MODS[-1]} +map ${MAP_BASE_NAME}
```
(What this evaluates to is something like... `/path/to/vkquake -basedir /path/to/quakedev -game id1 +map my_map`)

I had based my vkquake on the quakespasm settings and copied those args verbatim. However, removing all but `+map` fixed this issue for me:
```
+map ${MAP_BASE_NAME}
```

All that being said.... I wouldn't expect that providing those extra args would break behavior.
